sdk/identity/Azure.Identity/src/MsalClientBase.cs

```diff
@@ -78,5 +83,18 @@ protected void LogMsal(LogLevel level, string message, bool isPii)
                 AzureIdentityEventSource.Singleton.LogMsal(level, message);
             }
         }
+
+        protected void LogAccountDetails(AuthenticationResult result)
+        {
+            if (_logAccountDetails)
+            {
+                string clientId = null;
+                if (result?.ClaimsPrincipal?.Identity is ClaimsIdentity claimsIdentity)
+                {
+                    clientId = claimsIdentity.Claims.FirstOrDefault(c => c.Type == "aud")?.Value;
```

Comment:
I believe `aud` is short for audience which isn't the same as client id, rather it's the service the token will be presented to.